src/Kmd.Logic.Gateway.Automation/Client/Extensions/GatewayClient.ProductExtensions.cs

```diff
@@ -642,7 +642,8 @@ public async Task<HttpOperationResponse<object>> CustomUpdateProductWithHttpMess
                 responseContent = await httpResponse.Content.ReadAsStringAsync().ConfigureAwait(false);
                 try
                 {
-                    result.Body = SafeJsonConvert.DeserializeObject<IDictionary<string, IList<string>>>(responseContent, this.DeserializationSettings);
+                    result.Body = SafeJsonConvert.DeserializeObject<string>(responseContent, this.DeserializationSettings);
+                    Console.WriteLine(result.Body);
```

Comment:
Should we print this to output?

Reply:
We are displaying the appropriate error message. If we are not printing this, then the error message is not displayed in console

Reply:
If we print this , we will not be able to figure out for which entity (product) we get error.I think we should make use of "GatewayAutomationResult" to write errors

Reply:
Removed Console.writeline